recipe/meta.yaml

```diff
@@ -8,14 +8,17 @@ source:
   fn: gsl-{{ version }}.tar.gz
   url: http://ftpmirror.gnu.org/gsl/gsl-{{ version }}.tar.gz
   sha256: 13d23dc7b0824e1405f3f7e7d0776deee9b8f62c62860bf66e7852d402b8b024
+  
+  patches:                         # [win]
+  # https://github.com/ampl/gsl/  
+    - patch.diff                   # [win]
```

Comment:
I should explain. Patches directory contains individual commit patches, the recipe contains a single patch covering all commits. Having individual pathces might be easier to read, but I couldn't make them work. So, I guess, I should remove the whole directory, `patches/`.